nearcore/src/config.rs

```diff
@@ -107,8 +107,8 @@ pub const NUM_BLOCKS_PER_YEAR: u64 = 365 * 24 * 60 * 60;
 /// Initial gas limit.
 pub const INITIAL_GAS_LIMIT: Gas = 1_000_000_000_000_000;
 
-/// Initial gas price.
-pub const MIN_GAS_PRICE: Balance = 1_000_000_000;
+/// Initial and minimum gas price.
```

Comment:
Could you expand this a bit? I want to capture that this is *default* initial price -- what we use in tests and sandbox. The actual price is whatever is in config of a particular chain. Maybe even rename it to `DEFAULT_MIN_GAS_PRICE`?

Reply:
That was exactly my thought process as well. I already changed the name to exactly your suggestion and wrote a comment. But then I realised that this is true for all the constants in this file that relate to genesis. They are all used as default values when generating a new genesis file. So it felt a bit out of place to add specific comments to this particular constant. So I removed my comment and thought it might be better to leave it as it is.

Clearly this is somewhat confusing still. But I don't know where best to add this information. Prefixing all the constants with `DEFAULT_` seems a bit verbose. Adding a comment to each constant also seems verbose. But do you think it would be worth it?

Reply:
:thinking: I think we might want to do an overhaul of `GenesisConfig`. We have too many defaults there: 

* SmartDefault
* serde defaults
* these constants

I think ideally we'd do a manual `impl Default` which just sets all of the constants, and then adjust everything else to build on that, but this is out of scope for the PR

Reply:
Yeah - we should redesign how we're handing genesis configs. As currently we roughly need to cover 3 scenarios:

* mainnet/testnet - where we can no longer change the 'original' file
* sandbox - where we can setup whatever we want - but it should match mainnet/testnet behavior
* localnet/shardnet etc - where can setup whatever we want - and doesnt' have to match mainnet/testnet behavior.

We have a lot of code to handle the case number 1  (the code with MIN_GAS_PRICE_NEP_92_FIX is a great example of this).

I'd actually suggest, that for case 1) - we do something similar to what we do for gas pricing. By having a 'view' of the genesis_$PROTOCOL_VERSION_mainnet.json and genesis_$PROTOCOL_VERSION_testnet.json somewhere in the code..

But that's outside of this PR scope.